check.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -ex
```

Comment:
```suggestion
#!/bin/bash

set -euxo pipefail
```
`-u` ensures you don't use any unset variables, and `-o pipefail` ensures that if a command in a pipe fails, the overall command still exits with an error.

Reply:
Thanks!

I learnt something new today about bash :D